cmd.sh

```diff
@@ -2,29 +2,15 @@
 
 set -e
 
-cat > body <<EOF
-{
-  "title": "$title",
-  "body": "$body",
-  "head": $head,
-  "base": $base
-}
-EOF
-
 statusCode=$(curl \
     --silent \
-    --output response \
+    --output /pullRequest.json \
     --write-out "%{http_code}" \
     --user "${loginUsername}:${loginPassword}" \
     -X POST "https://api.github.com/repos/${owner}/${repo}/pulls" \
-    -d @body)
-
-echo "$statusCode"
-cat response
+    -d @/request.json)
 
 if test "$statusCode" -ne 201; then
+cat /pullRequest.json
```

Comment:
nit: would be nice to have something like
```suggestion
echo "failed to create pull request for repo $repo. Response code: $statusCode"
cat /pullRequest.json
```